src/test/java/com/wikia/webdriver/common/core/drivers/browsers/ChromeBrowser.java

```diff
@@ -0,0 +1,82 @@
+package com.wikia.webdriver.common.core.drivers.browsers;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import com.wikia.webdriver.common.core.ExtHelper;
+import com.wikia.webdriver.common.core.WikiaWebDriver;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.drivers.BrowserAbstract;
+import com.wikia.webdriver.common.driverprovider.UserAgentsRegistry;
+
+public class ChromeBrowser extends BrowserAbstract {
+
+  private ChromeOptions chromeOptions = new ChromeOptions();
+  private boolean useMobile = "CHROMEMOBILEMERCURY".equals(Configuration.getBrowser());
+
+  @Override
+  public void setOptions() {
+    String chromeBinaryPath = "";
+    String osName = System.getProperty("os.name").toUpperCase();
+    String emulator = Configuration.getEmulator();
+
+    if (osName.contains("WINDOWS")) {
+      chromeBinaryPath = "/chromedriver_win32/chromedriver.exe";
+    } else if (osName.contains("MAC")) {
+      chromeBinaryPath = "/chromedriver_mac32/chromedriver";
+
+      File chromedriver =
+          new File(ClassLoader.getSystemResource("ChromeDriver" + chromeBinaryPath).getPath());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "ChromeDriver" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)